v2/internal/pkg/config/validate.go

```diff
@@ -18,50 +19,92 @@ var validationDeleteChecks = []validationDeleteFunc{validateOperatorOptionsDelet
 func Validate(cfg *v2alpha1.ImageSetConfiguration) error {
 	var errs []error
 	for _, check := range validationChecks {
-		if err := check(cfg); err != nil {
-			errs = append(errs, fmt.Errorf("invalid configuration: %v", err))
+		if validationErrs := check(cfg); len(validationErrs) > 0 {
+			errs = append(errs, fmt.Errorf("invalid configuration: %v", utilerrors.NewAggregate(validationErrs)))
```

Comment:
What is the difference between this utilerrors and errors.Join from the standard library? 

If the goal is the same for both packages, we could reduce one package dependency. But only if the effort to change it is really small.